common/models/acl.js

```diff
@@ -546,27 +561,32 @@ module.exports = function(ACL) {
    * @param {Function} cb Callback function
    */
   ACL.isMappedToRole = function(principalType, principalId, role, cb) {
+    cb = cb || utils.createPromiseCallback();
```

Comment:
here, i'm turning ACL.isMappedToRole() promisable with loopback utils. In order to ease the change (managing the `return`) and keep the flow readable i refactored using promises inside the method definition. Is that OK? very few part of the code excepting tests actually use promises in methods